appveyorDashboard.yml

```diff
@@ -0,0 +1,15 @@
+install:
```

Comment:
Is there an AppVeyor build that has succeeded with these changes? I see a couple projects, but both look like they're failed or not running tests?